addons/web/static/src/search/search_panel/search_view.scss

```diff
@@ -20,6 +20,13 @@
         background: inherit;
     }
 
+    .o_searchview_facet {
+        cursor: default;
+        &.o_facet_with_domain:hover .o_searchview_facet_label {
+            background-color: green;
```

Comment:
I don't like the green color and there is a problem with the facet of favorites. Do you have a solution for that?

Reply:
The idea of the whole thing is to make the user easily understand that the label is clickable for a facet with domain.